src/store/index.ts

```diff
@@ -177,7 +184,22 @@ const {
       if (networkTables === undefined) {
         return;
       }
+      // Check node and table size
```

Comment:
It's a bit cleaner if we add a line of white space above this block
```suggestion
     
      // Check node and table size
```